tools/build-base-images.sh

```diff
@@ -19,7 +19,14 @@
 
 set -ex
 
-HUB="${HUB:-istio.io/docker}"
+HUB="${HUB:?specify a hub}"
 TAG="${TAG:?specify a tag}"
 
+# For multi architecture building:
+# See https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408 for more info
+# * docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
consider use this instead.
```
docker run --privileged --rm tonistiigi/binfmt --install all
```
https://github.com/docker/buildx#building-multi-platform-images

`tonistiigi/binfmt:latest` is a multi arch image with qemu@5.0.0 (multiarch/qemu-user-static only support amd64, and with qemu@4)

Reply:
Thanks!